templates/web_log.conf.j2

```diff
@@ -10,7 +10,8 @@ autodetection_retry: {{ openio_netdata_python_d_retry }}
     name: '.openio.{{ openio_netdata_namespace }}.rawx.rawx-{{ loop.index0 }}.log.access'
     path: '/var/log/oio/sds/{{ openio_netdata_namespace }}/rawx-{{ loop.index0 }}/rawx-{{ loop.index0 }}-httpd-access.log'
```

Comment:
The path is not good

Reply:
We are keeping the legacy file format for this case, the change has been ported to rsyslog.